src/components/molecules/AdminInput/AdminInput.tsx

```diff
@@ -21,6 +22,7 @@ export const AdminInput: React.FC<AdminInputProps> = ({
   subtext,
   register,
   errors,
+  hidden,
```

Comment:
```suggestion
  hidden: isHidden,
```
something like this to match the code style

Reply:
done @ 1f8dfd85d82d90d00f2a901db744bd5ba1ea6304